plugins/cilium-cni/chaining/generic-veth/generic-veth.go

```diff
@@ -61,7 +62,7 @@ func (f *GenericVethChainer) Add(ctx context.Context, pluginCtx chainingapi.Plug
 	defer ns.Close()
 
 	if err = ns.Do(func() error {
-		links, err := netlink.LinkList()
+		links, err := safenetlink.LinkList()
```

Comment:
FWIW (I'll look in to this) I think we can get rid of this LinkList call anyways. It should always be the interface specified by CNI_IFNAME.